README.md

```diff
@@ -119,78 +118,56 @@ cargo build -p millau-bridge-node
 cargo build -p substrate-relay
 ```
 
-### Running
+### Running a Dev network
+
+We will launch a dev network to demonstrate relay from millau to rialto here
+
+#### Running from local scripts
 
 To run a simple dev network you'll can use the scripts located in
-[the `deployments/local-scripts` folder](./deployments/local-scripts). Since the relayer connects to
-both Substrate chains it must be run last.
+[the `deployments/local-scripts` folder](./deployments/local-scripts). Since the relayer connects to both Substrate chains it must be run last.
```

Comment:
Can you make this two lines again? We wrap our files at 100 characters (see the [`.editorconfig`](https://github.com/paritytech/parity-bridges-common/blob/master/.editorconfig#L9)).